src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/Cohort/ReviewApprenticeRequestToReviewApprenticeViewModelMapper.cs

```diff
@@ -41,7 +41,8 @@ public async Task<ReviewApprenticeViewModel> Map(ReviewApprenticeRequest source)
             var csvRecords = await _cacheService.GetFromCache<List<CsvRecord>>(source.CacheRequestId.ToString());
             _logger.LogInformation("Total number of records from cache: " + csvRecords.Count);        
 
-            var groupedByCohort = csvRecords.Where(x => x.CohortRef == source.CohortRef );
+            var groupedByCohort = csvRecords.Where(x => x.CohortRef == source.CohortRef || 
```

Comment:
this var is called `groupByCohort but there is no grouping happening here only filtering?